selfdrive/ui/qt/widgets/controls.cc

```diff
@@ -47,7 +46,6 @@ AbstractControl::AbstractControl(const QString &title, const QString &desc, cons
     description->setWordWrap(true);
     description->setVisible(false);
     vlayout->addWidget(description);
-    vlayout->addStretch();
```

Comment:
Without this stretch, the title will adjust the position and flash during folding

Reply:
I don't think this is the real solution, it breaks the centering